src/patternfly/components/Table/table.scss

```diff
@@ -975,6 +975,10 @@
   vertical-align: middle;
 }
 
+.pf-c-table__action {
+  text-align: right;
```

Comment:
I agree we should push the stuff to the right, but this could cause 2 problems

* `text-align` is inherited, so it is going to apply to all children. looks like we set `text-align` in our menu items since we use buttons and those are center aligned by default.
* It could cause an unexpected layout change users might need to address when they pull in this version. As common/integral as the table is, I think we should wait for a breaking change to change the alignment.

wdyt?

Reply:
@mcoker wfm, captured here: https://github.com/patternfly/patternfly/issues/4661